### ISSUE TITLE
transpose does not suppport  quant

### ISSUE THREAD

**Issue by sunzhe09** [tensorflow/models#8283]
**System information**
- Have I written custom code (as opposed to using a stock example script provided in TensorFlow):
- OS Platform and Distribution (e.g., Linux Ubuntu 16.04):16.04
- Mobile device (e.g., Pixel 4, Samsung Galaxy 10) if the issue happens on mobile device:
- TensorFlow installed from (source or binary):
- TensorFlow version (use command below):1.14.0
- Python version:
- Bazel version (if compiling from source):
- GCC/Compiler version (if compiling from source):
- CUDA/cuDNN version:
- GPU model and memory:

**Please provide the entire URL of the model you are using?**

**Describe the current behavior**
toco run failed,axis is much big or negative number
**Describe the expected behavior**
the axis should be (0,3)
**Code to reproduce the issue**

tf.squeeze()+tf.transpose() will occur the bug.
if I use tf.squeeze()+tf.reshape() it pass

**Other info / logs**

2020-03-13 11:19:52.146016: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] Before Removing unused ops: 2359 operators, 3524 arrays (0 quantized)
2020-03-13 11:19:52.372475: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] Before general graph transformations: 2359 operators, 3524 arrays (0 quantized)
2020-03-13 11:19:52.982458: I tensorflow/lite/toco/graph_transformations/graph_transformations.cc:39] After general graph transformations pass 1: 342 operators, 672 arrays (1 quantized)
2020-03-13 11:19:52.984716: F tensorflow/lite/toco/graph_transformations/propagate_fixed_sizes.cc:1813] Check failed: axis < input_shape.dimensions_count() (1696750784 vs. 4)
Fatal Python error: Aborted

**Comment by ravikyram** [comment]
@sunzhe09 

It is difficult to understand your issues only based on the information you provided.Please, let us know which pretrained model you are using? .Thanks!

**Comment by ravikyram** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
